new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js

```diff
@@ -80,8 +81,10 @@ const BooleanPropertiesTable = memo(
             {radioGroupValues &&
               radioGroupValues.map((element, idx) => (
                 <TableRow key={idx} size="sm" className={classes.tableRow}>
-                  <TableCell className={classes.tableCell}>
+                  <TableCell className={classes.leftTableCell}>
                     {element.display}
+                  </TableCell>
+                  <TableCell className={classes.rightTableCell}>
                     {editing ? (
                       <RadioGroup
                         options={radioButtonOptions}
```

Comment:
I don't see this variable anywhere.

Reply:
It's [here](https://github.com/liordino/lamassu-server/blob/feature/add-receipt-printing-screen/new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.styles.js#L34).

Reply:
I mean this one: `options={radioButtonOptions} `

Reply:
Ow, sorry. Anyway, isn't it [this one](https://github.com/lamassu/lamassu-server/blob/fd4924d3c832f2439cd5328c151b0c7f0c668ba7/new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js#L48)? It's still wrong anyway, because it's yet using label instead of display. I'll fix it today.

Reply:
Fixed. Had to also refactor label to display on the ContactInfo page.